src/hotspot/share/opto/block.cpp

```diff
@@ -395,6 +395,9 @@ PhaseCFG::PhaseCFG(Arena* arena, RootNode* root, Matcher& matcher)
   x->init_req(0, x);
   _goto = matcher.match_tree(x);
   assert(_goto != nullptr, "");
+  if (C->failing()) {
```

Comment:
The new flag found several missing `failing()` checks, which are added in this patch.